src/Pages/LabelPage/LabelPage.jsx

```diff
@@ -1,142 +1,132 @@
 import React from 'react'
 import "./LabelPage.css"
 
+
+import { useEffect, useState } from "react";
+import { useNotes } from '../../Context/noteContext';
+import { useAuth } from '../../Context/authContext';
+
 const LabelPage = () => {
+
+
+	const { getNotes } = useNotes();
+	const { authState } = useAuth();
+	const { notes } = authState;
+
+	const tags = ["all", "Home", "Work", "Personal"];
+
+
+	const [labels, setLabels] = useState({
+		all: true,
+		Home: false,
```

Comment:
instead of this you can create a one single state as currentLabel or selectedLabel and set initial value as "all"
and update it on click